From c1f9b6c4d82b53fa997b29d442f163b3ff8cba2e Mon Sep 17 00:00:00 2001
From: Artem Borisov <dedsa2002@gmail.com>
Date: Sat, 16 Feb 2019 11:36:33 +0400
Subject: [PATCH] Snap: Fix shutter button size

* This probably got messed up with some of the recent rebases,
  making our shutter button ugly and misaligned.
* Set it to 80dp which is the proper value for our drawables.

Change-Id: Ief53c8c121bc8a3ad1a81fe6780d78ca90bf39a9
---
 res/values/dimens.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values/dimens.xml b/res/values/dimens.xml
index 29fbbe561..415404d37 100755
--- a/res/values/dimens.xml
+++ b/res/values/dimens.xml
@@ -110,7 +110,7 @@
     <dimen name="capture_margin_top">16dip</dimen>
     <dimen name="camera_controls_size">0dip</dimen>
     <dimen name="camera_film_strip_gap">32dip</dimen>
-    <dimen name="shutter_size">68dp</dimen>
+    <dimen name="shutter_size">80dp</dimen>
 
     <dimen name="appwidget_width">180dp</dimen>
     <dimen name="appwidget_height">180dp</dimen>
-- 
2.17.1

